lib/hello_sign/client.rb

```diff
@@ -154,6 +154,7 @@ def make_connection options
         faraday.request :url_encoded
         faraday.response :logger, logger if @logging
         faraday.adapter :net_http
+        faraday.options[:timeout] = timeout if timeout
```

Comment:
I named it "timeout" instead of "read_timeout" because that's what Faraday calls it.